net-p2p/amule/amule-2.3.2-r4.ebuild

```diff
@@ -1,19 +1,19 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Also please don't use `git mv` or edit the previous file, but make a new copy, edit that, commit that+Manifest and if you wish, make a new commit removing the old file+Manifest hash.

Reply:
done